src/utils/strings.ts

```diff
@@ -9,6 +9,10 @@ const KEY_CASE_OVERWRITE = {
   url: 'URL',
 };
 
+type NullUndefined = null | undefined | unknown;
```

Comment:
how come unknown is included here?

Reply:
The type definition is `unknown` by default (that `setType` thing I mentioned). The type can be dynamically be set but by the consumer. So right now it would be `unknown`

Reply:
as discussed in slack, this was done to appease the linter